### ISSUE TITLE
Blender 4.0 Screencast Addon not working

### ISSUE THREAD

**Issue by Maakiets** [nutti/Screencast-Keys#187]
Screencast Keys is nog working in Blender 4.0. 

Is there any change there wil be a new version? I am in the middle of recordig a Dutch training course and I need the screencast key's 

Thanks!

Bart Valkenier 

Maakiets Media

**Comment by nutti** [comment]
@Maakiets 

Any update?

**Comment by Maakiets** [comment]
Yes! Still unable to use the addon in Blender 4.0. I have 3 video's showing the problem. 1- the current problem 2- same problem after reinstalling Blender 4.0 and the add on 3- same problem after restetting Blender 4.0 and reinstalling the addon. 

Thanks for this great addon and your support, hopfully I am doing something stupid an is a quick fix. 

Thanks, 

Bart Valkenier 
Maakiets Media 

https://github.com/nutti/Screencast-Keys/assets/151615282/4b388ad4-20e9-4f23-9227-011479731795

https://github.com/nutti/Screencast-Keys/assets/151615282/3693e7e2-ce24-4e41-b9fb-b24e921b2f4c

https://github.com/nutti/Screencast-Keys/assets/151615282/b779ed08-9183-43a8-8117-cbbcdc33ac76

**Comment by nutti** [comment]
@Maakiets 

Could you paste your error message on this issue?
The resolution of the movie 3 is bad, so I could not recognize the error.

**Comment by Maakiets** [comment]
It works now! I restarted Blender, tip form issue #188  and now it works! So I can't screengrab the error massage anymore but again, it works

Thanks!

**Comment by nutti** [comment]
@Maakiets 

I'm glad you solve the issue.
